tests/test_stl.py

```diff
@@ -326,3 +326,18 @@ def test32_list():
     assert t.identity_list([1, 2, 3]) == [1, 2, 3]
     assert t.identity_list(()) == []
     assert t.identity_list((1, 2, 3)) == [1, 2, 3]
+
+def test33_string_and_string_view():
+    assert t.identity_string("") == ""
+    assert t.identity_string("orange") == "orange"
+    assert t.identity_string("橘子") == "橘子"
+    assert t.identity_string("ส้ม") == "ส้ม"
+    assert t.identity_string("البرتقالي") == "البرتقالي"
```

Comment:
Is there something funny going on with parentheses here?

Reply:
I don't quite understand. Is there anything wrong with the parentheses?

Reply:
Could it be a rendering issue? At least on the GitHub web interface, I don't see a closing perenthesis after the arabic strings.

Reply:
This is what I see:
<img width="596" alt="image" src="https://user-images.githubusercontent.com/3606672/192822398-46e9c196-0fe4-4ba9-b7d6-ce5c82b86353.png">